packages/ag-charts-website/src/features/api-documentation/apiReferenceHelpers.ts

```diff
@@ -87,14 +87,20 @@ export function normalizeType(refType: TypeNode, includeGenerics?: boolean): str
     }
 }
 
-export function processMembers(interfaceRef: InterfaceNode | TypeLiteralNode | EnumNode, config: ApiReferenceConfig) {
+export function processMembers(
+    interfaceRef: InterfaceNode | TypeLiteralNode | EnumNode,
+    config: ApiReferenceConfig,
+    prefixPath: string[]
+) {
     let { members } = interfaceRef;
+    const prefix = prefixPath.join('.');
     const { prioritise, include, exclude } = config;
     const isInterface = interfaceRef.kind === 'interface';
     if (include?.length || exclude?.length) {
-        members = members.filter(
-            (member) => !exclude?.includes(member.name) && (include?.includes(member.name) ?? true)
-        );
+        members = members.filter((member) => {
+            const fullMemberName = prefix !== '' ? `${prefix}.${member.name}` : member.name;
+            return exclude?.includes(fullMemberName) !== true && include?.includes(fullMemberName) !== false;
```

Comment:
nitpick: Couldn't you remove the boolean comparison?

```suggestion
            return !exclude?.includes(fullMemberName) && include?.includes(fullMemberName);
```